raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -108,7 +92,7 @@ library ChannelManagerLibrary {
         address channel_address,
         uint caller_index,
         uint partner_index)
-        private
+        internal
```

Comment:
The visibility was changed here so that the `deleteChannel()` wrapper function in `ChannelManagerContract` can call the `deleteChannel()` function in the `ChannelManagerLibrary`